### PR TITLE
Add multi domain certificate ability

### PR DESCRIPTION
**Pull request by matclab**
Add ability to generate certificate for multi domains.

User as to manually add domains on the command line, as a comma separated list.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -129,8 +131,28 @@ def _b64(b):
         domain_key_file.write(domain_key_contents)
 
       log.info("Generate csr...")
-      proc = subprocess.Popen(["openssl", "req", "-new", "-sha256", "-key", domain_key, "-subj", "/CN=" + domain],
-                              stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+      if len(domains) > 1:
+        domain_config_file = tempfile.NamedTemporaryFile()
+        default_conf = open("/etc/ssl/openssl.cnf").readlines()
```
Comment (andrin-n-dream) [inline __init__.py:137]: Is there a way you can make this platform independent? /etc/sll/openssl.cnf only works on Linux ...

**Comment by andrin-n-dream** [comment]
Also document the change on line 55.

**Comment by andrin-n-dream** [comment]
The rest looks good! Thanks for the PR

File: __init__.py (hunk 1 above)
Comment (matclab) [inline __init__.py:137]: Alas, I really do not know what is needed or not in the conf file.
The best I can do is to inline a copy of the openssl.cnf file in the script, but I don't know if we want it or not.

Also, on Windows the NamedTemportaryFile should not be openned by the openssl command without being closed before in the script (and thus, created with the `delete = False` parameter, and being deleted manually after the run of openssl.

Comment (matclab) [inline __init__.py:137]: Also, `domain` shall be replaced by `domainlist` in the case where there is only one domain.

**Comment by matclab** [comment]
@andrin-n-dream, you mean "document the change line 205", about the agreement (like in a separate commit) ? Or something else ?

**Pull request merged**